backend/src/main/java/com/woowacourse/gongseek/comment/application/CommentService.java

```diff
@@ -27,15 +27,15 @@ public class CommentService {
 
     public void save(AppMember appMember, Long articleId, CommentRequest commentRequest) {
         validateGuest(appMember);
-        Member member = findMember(appMember);
-        Article article = findArticle(articleId);
+        Member member = getMember(appMember);
+        Article article = getArticle(articleId);
         Comment comment = new Comment(commentRequest.getContent(), member, article);
 
         commentRepository.save(comment);
     }
 
     @Transactional(readOnly = true)
-    public CommentsResponse findByArticleId(AppMember appMember, Long articleId) {
+    public CommentsResponse getAll(AppMember appMember, Long articleId) {
```

Comment:
```suggestion
    public CommentsResponse getByArticleId(AppMember appMember, Long articleId) {
```
위와 같은 메서드명은 어떤가요?
모든 댓글을 가져오는게 아니고 하나의 게시물에 대한 댓글을 가져온다는 것을 명확하게 나타내면 좋을 것 같아요.

Reply:
get 이라고 하면 하나만 가져오는 느낌이 들어서 getAllByArticleId로 수정했습니다. ByArticleId로 해당 게시글에 대한 댓글을 모두 가져온다는 의미를 전달할 수 있을 것같아요.